app/controllers/all-service-transactions/get.controller.js

```diff
@@ -20,6 +22,15 @@ module.exports = async (req, res) => {
   try {
     const userPermittedAccountsSummary = await permissions.getLiveGatewayAccountsFor(req.user, 'transactions:read')
 
+    const logContext = {
+      gateway_account_ids: userPermittedAccountsSummary.gatewayAccountIds,
+      user_number_of_live_services: userPermittedAccountsSummary.gatewayAccountIds && userPermittedAccountsSummary.gatewayAccountIds.length,
```

Comment:
We already have this number on the User object. Wonder whether we should use that so that if we end up loging this in multiple places it can be uniform https://github.com/alphagov/pay-selfservice/blob/master/app/models/User.class.js#L53

Reply:
Didn't realise we have this field on User object. Updated to use User property